src/styles/blocks/header.scss

```diff
@@ -0,0 +1,71 @@
+.header {
+  background-color: #3f5252;
+
+  padding-top: 20px;
+  margin-bottom: 90px;
+
+  @include onTablet {
+    padding-top: 26px;
+    margin-bottom: 100px;
+  }
+
+  @include onDesktop {
+    padding-top: 26px;
+    margin-bottom: 110px;
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    color: #f1f5f4;
+  }
+
+  &__text {
+    margin-bottom: 10px;
+
+    font-size: 14px;
+    line-height: 140%;
+  }
+
+  &__title {
+    margin-bottom: 30px;
+
+    @include onTabletOnly {
+      margin-bottom: 20px;
+      font-size: 72px;
+    }
+
+    @include onDesktop {
+      font-size: 94px;
+    }
+  }
+
+  &__subtitle {
+    margin-bottom: 40px;
+    color: #f1f5f4;
+
+    @include onTablet {
+      margin-bottom: 50px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 80px;
+    }
+  }
+
+  &__img {
+    display: block;
+    bottom: 0;
+    height: 273px;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
Try to use shorthand properties
https://developer.mozilla.org/en-US/docs/Web/CSS/margin#constituent_properties